plugin/tokudb-backup-plugin/tokudb_backup.cc

```diff
@@ -29,6 +29,7 @@
 #endif
 
 static char *tokudb_backup_plugin_version;
+static PSI_memory_key tokudb_backup_mem_key = 0;
```

Comment:
I'm not sure I like this approach that creates what appears to be a memory key, yet is not a memory key. Readers at my_*alloc call will be led to thinking that the calls are annotated. Either go all the way already (it's ~5 more lines of code + unknown amount of MTR re-records for PFS), either just use PSI_UNINSTRUMENTED (sic?)